frontend/src/features/properties/components/forms/subforms/EvaluationForm.tsx

```diff
@@ -112,12 +112,13 @@ export const validateFinancials = (
   financialYears.forEach((financialYear, index) => {
```

Comment:
I think we may be at the point where literally none of this validation is required anymore.

Reply:
Confirm this with Mikaela, but we don't show appraisal anymore so that validation should not be required, and then I think netbook was the only field left that had any validation on it. If we aren't validating netbook anymore I don't think we are validating anything.

Reply:
Essentially we don't want to block them from saving the property.   There is no validation now on financial values (other than perhaps they are valid positive numbers 0-infinity)

Reply:
Ok that's what I was thinking originally was just wondering if I should have left it incase the requirements change again. But it's easy enough to add back in so removal is probably the best option.